src/modelfree/shaping_wrappers.py

```diff
@@ -186,8 +153,30 @@ def load(self):
         raise NotImplementedError()
 
 
+def load_wrapper_params(params_path, env_name, rew_shaping=False, noisy_victim=False):
+    config_dir = 'rew_configs' if rew_shaping else 'noise_configs'
```

Comment:
Should either add an assertion that `rew_shaping` and `noisy_victim` are not both `True`, or just take in directly a `config_dir` directory and check it is either `rew_configs` or `noise_configs` (then change behaviour later based on which of these it is ).